opensea-operator-filter/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "OpenseaOperatorFilter",
+  "scripts": {
+    "compile": "hardhat compile",
+    "typechain": "hardhat typechain",
+    "test": "hardhat test",
+    "prettier": "prettier --write 'contracts/**/*.sol' 'scripts/**/*.ts' 'test/**/*.ts'",
+    "deploy": "hardhat run scripts/deploy.ts --network"
+  },
+  "devDependencies": {
+    "@nomicfoundation/hardhat-chai-matchers": "^1.0.5",
+    "@nomicfoundation/hardhat-toolbox": "2.0.0",
+    "dotenv": "16.0.3",
+    "hardhat": "2.12.3",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.9.4"
+  },
+  "dependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.2.1",
+    "@nomiclabs/hardhat-etherscan": "^3.1.4",
+    "@openzeppelin/contracts": "4.8.0",
```

Comment:
In Bunzz repo, we use @openzeppelin/contracts ^4.3.2
So contracts should work using that version
@cc @hitsuji-haneta @deep-quality-dev @solcmoon

Reply:
@Top-Kraken 
fixed